virttest/utils_v2v.py

```diff
@@ -238,6 +238,9 @@ def _compose_vmx_filename():
                             ptn, self._nfspath).group(1)
                         break
 
+                if '%' in self._nfspath:
+                    self._nfspath = self._nfspath.replace('%', '%25')
+
```

Comment:
I prefer to update the '_vmx_filename'.
self._vmx_filename = (self._vmx_filename + '.vmx').replace('%', '%25')

Reply:
Ignore my previous comment. I was wrong. 
The _nfs_path is also used at other place.